tools/dsal/dsal_helper/templates/anaconda-ks-8.4.cfg.tpl

```diff
@@ -0,0 +1,429 @@
+#version=RHEL8
```

Comment:
there is a bunch of Red Hat infos in there, I'm not sure we want to put that on github?

Reply:
Aw, so this is a public repo? I guess I could encrypt it somehow?